cex-collection/js-example/.env

```diff
@@ -5,6 +5,8 @@ PAYMASTER_URL="https://bsc-megafuel-testnet.nodereal.io"
 SPONSOR_URL="https://open-platform-ap.nodereal.io/{YOUR_API_KEY}/megafuel-testnet"
 
 POLICY_UUID="xxxxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxx"
```

Comment:
`PRIVATE_POLICY_UUID` and `POLICY_UUID` just keep one

Reply:
fixed, keep PolicyUUID  for js-sdk & go-sdk